frontend/src/scenes/insights/InsightTabs/TrendTab/TrendTab.tsx

```diff
@@ -114,11 +113,13 @@ export function TrendTab({ view }: TrendTabProps): JSX.Element {
                     )}
                     {filters.insight !== InsightType.LIFECYCLE && (
                         <>
-                            {featureFlags[FEATURE_FLAGS.AND_OR_FILTERING] ? (
+                            {filters.properties && (
```

Comment:
Note: we'll want to keep this feature flag before merging

Reply:
The usual reasons of maintaining phased rollout control and easy feature shut off if things are buggy